evalml/preprocessing/utils.py

```diff
@@ -22,7 +22,7 @@ def load_data(path, index, target, n_rows=None, drop=None, verbose=True, **kwarg
         verbose (bool): If True, prints information about features and target
 
     Returns:
-        pd.DataFrame, pd.Series: features and target
+        Union[ww.DataTable, ww.DataColumn]: Features matrix and target
```

Comment:
Could this be just `ww.DataTable, ww.DataColumn` since we're returning two separate things? 😄